### ISSUE TITLE
support tpm

### ISSUE THREAD

**Issue by albasili** [siadat/session-finder#5]
It seems to me that your installation procedure is quite manual, it would be nice if this plugin supported tpm in order to integrate nicely with an already established flow.

**Comment by siadat** [comment]
Good idea, I had to have a look at tpm. It would also be great if you are interested in sending a PR. Thanks!

**Comment by axelson** [comment]
I've been using this project with TPM for a while now. Here's the patch I have: https://github.com/axelson/session-finder/commit/9ff919e7d7837766b9e64f287f9d239c2a708b8b

To make it a PR I guess you'd mainly need an accompanying documentation update

**Comment by siadat** [comment]
@axelson That looks good. It seems like a minor change, let me know if you wrote the accompanying documentation
